gnuboardProject/frontend/src/components/Profile/Profile.tsx

```
@@ -127,40 +245,84 @@ const Profile: React.FC = () => {
+    }
+  };
+
+  const handleEditChange = (e: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+    const { name, value } = e.target;
+    setEditProfile(prev => ({ ...prev, [name]: value }));
+  };
+
+  const handleImgChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      setEditProfile(prev => ({ ...prev, profileImage: e.target.files![0] as File }));
+      setImgPreview(URL.createObjectURL(e.target.files[0]));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)